hamcrest/Hamcrest/Type/IsNumeric.php

```diff
@@ -19,10 +19,31 @@ public function __construct()
 
     public function matches($item)
     {
+        if ($this->isHexadecimal($item)) {
+
```

Comment:
Are you sure, that this also applies to `return` that is the only statement in a scope (the `{` and `}`)?